app/controllers/pics_controller.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+class PicsController < ApplicationController
+  def index
+    authenticate_user!
+    @pics = Pic.order("created_at DESC")
+  end
+
+  def new
+    unless user_signed_in?
+      flash[:notice] = "You must be logged in to do that"
+      redirect_to pics_path
+    end
+    @pic = Pic.new
+  end
+
+  def create
+    pic = Pic.create(pic_params.merge(user_id: current_user.id))
```

Comment:
I'm getting this error when trying to create a new pic:

PG::UndefinedTable: ERROR: relation "active_storage_blobs" does not exist LINE 8:

Reply:
fixed